sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -56,6 +55,11 @@
   // Handler called when the span starts and ends.
   private final SpanProcessor spanProcessor;
   // The displayed name of the span.
+  // List of recorded links to parent and child spans.
+  private final List<Link> links;
```

Comment:
Nit: you can also update https://github.com/open-telemetry/opentelemetry-java/blob/43bd4a2993951e311f0cd42921e430c60f5656d6/sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java#L195 since it's no longer Nullable.

Reply:
Changed to !isEmpty to avoid creating unnecessary proto if empty.